schemas/test/en/test_introduction_with_guidance.json

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
Since we are trying to reduce schema, I think we can scrap this and add guidance to the existing intro schema. It would be good to make it looks like the one attached to the card, with content above and below the guidance.

Reply:
Moved to single schema now.